AA eviction clarification (#1913)
RedisLabs/redislabs-docs: content/rs/databases/configure/eviction-policy.md

`volatile-lru` is the default eviction policy for most databases.

The default policy for [Active-Active databases]({{< relref "/rs/databases/active-active/_index.md" >}}) is _noeviction_ policy.

## Active-Active database eviction
The eviction policy mechanism for Active-Active databases kicks in earlier than for regular databases because it requires propagation to all participating clusters. The eviction policy starts to evict keys when one of the Active-Active instances reaches 80% of its memory limit. If memory usage continues to rise while the keys are being evicted, the rate of eviction will increase to prevent reaching the Out-of-Memory state.
Like in Redis database, eviction of Active-Active is calculated per shard.

> Based on internal heuristics, keys might get evicted only when reaching 100% of shard's memory limit and not get evicted although the shard has reached 80% of its memory limit to prevent over eviction. 

Author: ```suggestion
To prevent over eviction, internal heuristics might prevent keys from being evicted when the shard reaches the 80% memory limit.  In such cases, keys will get evicted only when shard memory 100%.
```

> To prevent over eviction, internal heuristics might prevent keys from being evicted when the shard reaches the 80% memory limit.  In such cases, keys will get evicted only when shard memory 100%.

Reviewer: ```suggestion
To prevent over eviction, internal heuristics might prevent keys from being evicted when the shard reaches the 80% memory limit.  In such cases, keys will get evicted only when shard memory reaches 100%.
```
